### PR TITLE
Rework intro

### LINKED ISSUE #41: Update intro

### PR DESCRIPTION
**Pull request by tschubotz**
Closes #41

### CODE REVIEW

File: docs/intro_01_assets.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ sidebar_label: Assets on Ethereum
...
 
-There are actually two different types of accounts on Ethereum: externally owned accounts and contract accounts, and it’s becoming clearer the latter is key to mass adoption. Join us below for  a comparison of externally owned accounts and contract accounts, followed by a walkthrough of the main features and advantages of building with the Gnosis Safe smart contracts. 
+There are actually two different types of accounts on Ethereum: externally owned accounts and contract accounts, and it’s becoming clearer the latter is key to mass adoption. Join us below for  a comparison of externally owned accounts and contract accounts, followed by a walkthrough of the main features and advantages of building with the Gnosis Safe smart contracts.
```
Comment (rmeissner) [inline docs/intro_01_assets.md:11]: 2 soaces after `Join us below for`

File: docs/intro_03_gnosis_safe.md [hunk 2]
```diff
@@ -4,24 +4,24 @@ title: The Gnosis Safe
...
 
 ### Advanced access management
 
-By allowing to add **Gnosis Safe Modules** to a Gnosis Safe it is possible to defined fine-grained access management. It is for example possible to define a module that can only be use to **recover access** to a Gnosis Safe under specific circumstances. A very popular version of this is the **Social Recovery Module**
+You can add **Gnosis Safe Modules** to your Gnosis Safe. Thereby it is possible to implement more fine-grained access management. For instance, it is possible to define a module that can only be use to **recover access** to a Gnosis Safe under specific circumstances. A popular version of this is the **Social Recovery Module**. A different example are **allowance modules** which allows owners of a Gnosis Safe to grant limited exeuction permission such as a daily limit to external accounts.
```
Comment (rmeissner) [inline docs/intro_03_gnosis_safe.md:19]: >A different example **are** ... which **allows** 

should they be either both plural or singular?

Comment (tschubotz) [reply docs/intro_03_gnosis_safe.md:19]: yea, question is whether the allows refers to example or modules :)

I'll change it.

**Comment by tschubotz** [comment]
> Do we want to leave out external usages of the safe contracts completely?

I don't remember why I removed it. -> re added.

**Pull request merged**